ebodac/src/main/java/org/motechproject/ebodac/template/PdfReportCTemplate.java

```diff
@@ -11,6 +12,8 @@
 
     public PdfReportCTemplate(OutputStream outputStream) {
         super(TEMPLATE_PATH, FIRST_PAGE_RECTANGLE, outputStream);
+        PdfReportCTemplate.TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 6);
```

Comment:
those variables are static, so if you will export any of ReportC values will be changed and will be use for all other reports